src/MissingDocumentException.php

```diff
@@ -0,0 +1,29 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * MissingDocumentException file
+ *
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         2.0.1
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\Elasticsearch;
+
+use Cake\Core\Exception\Exception;
+
+/**
+ * Exception raised when an Document could not be found.
```

Comment:
```suggestion
 * Exception raised when a Document could not be found.
```

Reply:
This is the change I mentioned in the other pull request.

Reply:
I get it. Master branch already have a MissingDocumentException, I've removed mine.